src/main/java/pl/touk/sputnik/Main.java

```diff
@@ -26,17 +30,32 @@ public static void main(String[] args) {
             commandLine = cliWrapper.parse(args);
         } catch (ParseException e) {
             printUsage(cliWrapper);
-            System.out.println(e.getMessage());
+            System.err.println(e.getMessage());
```

Comment:
Please don't change it to err. I want this message to be printed to out, even though I know it's an error.